src/stories/Library/slider/init-slider.js

```diff
@@ -1,3 +1,63 @@
+// When using the keyboard, and using the arrow controls, there is a mismatch
+// when you tab from the controls into the actual slider.
+// It always wants to add the focus to the first item in the slider,
+// meaning the slider will be reset.
+// We want to instead set focus on the first fully-visible slide.
+// However, we obviously don't want to set the focus on this, if you're moving
+// backwards (e.g. from slider => controls => whatever).
+// We get around this, by creating a "focus controller" - e.g. a custom element
+// that is created after the controls.
+// When focus is introduced to the focus controller, it gets removed, and we set
+// the focus to the first active slide.
+// We create the FocusController whenever the focus is set on the controls, so
+// that way, it's only when moving 'forwards' in the DOM tree that we hijack
+// focus.
+function createFocusController(swiper, controls) {
+  const focusControllerClass = "slider__focus-controller";
+  let focusController = controls.querySelector(`.${focusControllerClass}`);
+
+  // If a focus controller already exists, we won't create a new one.
+  if (focusController) {
+    return;
+  }
+
+  // Creating our custom focus element.
+  focusController = document.createElement("button");
+  focusController.classList.add(focusControllerClass);
+  controls.appendChild(focusController);
+
+  // When focus is set to the controller, we find the first visible slide,
+  // and set the focus to that.
+  // We also remove the focus controller, to make sure that we can tab
+  // backwards up the tree, without being hijacked.
+  focusController.addEventListener("focus", () => {
+    focusController.remove();
```

Comment:
From an accessibility perspective I worry about this approach. It creates an (invisible?) button without without any content/text. I wonder how this will behave for screen readers and the like.

Reply:
I get what you're saying, but, technically speaking, the screenreader wont even detect it, as, as soon as you focus it, it gets removed.

I'll ask Nino & co to be extra dilligent when testing the sliders, both in accessibility and general experience.